IM_calculation/IM/im_calculation.py

```diff
@@ -376,11 +376,7 @@ def get_bbseis(input_path, file_type, selected_stations):
             station_names = bb_stations
         else:
             # making sure selected stations are in bbseis
-            station_names = [
-                station_name
-                for station_name in selected_stations
-                if station_name in bb_stations
-            ]
+            station_names = selected_station.intersection(bb_stations)
```

Comment:
Isn't selected_station a list not a set?

Reply:
fixed